distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Thread.enso

```diff
@@ -19,3 +20,18 @@ import project.Any.Any
          Thread.with_interrupt_handler (1 + 1) <| IO.println "I died!"
 with_interrupt_handler : Any -> Any -> Any
 with_interrupt_handler ~action ~interrupt_handler = @Builtin_Method "Thread.with_interrupt_handler"
+
+## PRIVATE
+   ADVANCED
+   Temporarily cease execution of the current thread.
+
+   Arguments:
+   - time: amount of milliseconds to sleep
+
+   > Example
+     Sleep the current thread for 1 second.
+
+         Thread.sleep 1000 <| IO.println "I continue!"
+sleep : Integer
+sleep time_in_milliseconds = @Builtin_Method "Thread.sleep"
```

Comment:
First time I hear about this restriction. It's also not respected in other modules.

Reply:
Shouldn't be a `private` method think we want to have a sleep command available.

Reply:
> First time I hear about this restriction. 

You might have heard about builtin methods behaving strangely in various situations.

> It's also not respected in other modules.

No, it is not. Alas, the issue hasn't received enough priority to be fixed and generally enforced.

> I added a private next to ## PRIVATE as appears to be done in some other places.

- since #7840 by @Akirathan the preferred way to structure our code is to move `## PRIVATE` methods into `private` modules. Libraries team may not be completely consistent with it, although they are being reminded about it at every opportunity. 

> this can't be the reason to reject this PR

I'd like to understand why this is builtin and not just a `polyglot java import`? So far I see no reason for preferring builtin over `polyglot java import`. No benefit, just drawbacks (yet another builtin). These doubts convinced me to request changes - e.g. ask for an explanation and discuss the alternatives.

Reply:
> * since [Implement private modules #7840](https://github.com/enso-org/enso/pull/7840) by @Akirathan the preferred way to structure our code is to move `## PRIVATE` methods into `private` modules. Libraries team may not be completely consistent with it, although they are being reminded about it at every opportunity.

I don't see the point on insisting on tiny changes in a style that isn't enforced. Either we do it on one go and enforce or don't complain about it.

> I'd like to understand why this is builtin and not just a `polyglot java import`? So far I see no reason for preferring builtin over `polyglot java import`. No benefit, just drawbacks (yet another builtin). These doubts convinced me to request changes - e.g. ask for an explanation and discuss the alternatives.

Yes, I knew that adding a builtin will be problematic for you. Yet, my preference is to write more Enso in the libraries and staying within Enso as much as possible rather than relying on Java API, even if it means it just a wrapper around it. 

I really don't see the point of wasting my energy on this and I doubt I will manage to convince you so I reverted to plain java.lang.Thread import.